doc/source/rllib/package_ref/replay_buffers.rst

```diff
@@ -0,0 +1,132 @@
+.. _replay-buffer-api-reference-docs:
+
+Replay Buffer API
+==================
+
+RLlib comes with a set of extendable replay buffers which are used mostly in Q-Learning algorithms.
+We provide a base :py:class:`~ray.rllib.utils.replay_buffers.replay_buffer.ReplayBuffer` class from which you can build your own buffer.
+Some use cases may require :py:class:`~ray.rllib.utils.replay_buffers.multi_agent_replay_buffer.MultiAgentReplayBuffer`\s.
+You can find buffer types and arguments to modify their behaviour as part of RLlib's default parameters. They are part of
+the ``replay_buffer_config``.
+
+Basic Usage
+===========
+
+You rarely have to instantiate your own replay buffer when running an experiment, but rather configure it as follows.
+The following is from RLlib's examples section and runs the R2D2 algorithm, which runs without `PER <https://arxiv.org/abs/1511.05952>`__ by default, with PER:
+
+.. dropdown:: **Changing a replay buffer configuration**
+    :animate: fade-in-slide-down
+
+    .. literalinclude:: ../../../../rllib/examples/replay_buffer_api.py
+        :language: python
+        :start-after: __sphinx_doc_basic_replay_buffer_usage__begin__
```

Comment:
I'm having a hard time figuring out which part "activates" PER here. Can we either have a better documented snippet or create a "minimal" snippet that doesn't change too many things (model etc.)?

Reply:
I try to explain what users are going to see and what to look for before showing a snippet, but there are other ways.

Reply:
also, can we please add some imports here to make this snippet executable (if possible)?